packages/api/prisma/schema.prisma

```diff
@@ -69,7 +69,7 @@ model ArticleRevision {
 }
 
 model Article {
-  id         String   @id @default(cuid())
+  id         String   @id @default(uuid())
```

Comment:
I guess we can have default on postgresql but abitlity to set specific one. @Itrulia any thoughts on that? I actually thought about passing it from code.

Reply:
We probably should use the DB specific one ala https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#override-default-value-behavior-for-supported-types

Reply:
My issue with this change is more so that old articles are not migrated.

Reply:
Agree to Prisma internal. 

Yes, definitely we need migration in this PR too. 
As Karl suggested.
Making sure we have proper onUpdate cascades and setting to null. Hmm @Itrulia  won't this drop the relation when setting to `null` and then when re-adding it, it won't have the connections with other entities? I'm really not sure how this behaves. I would do some tests.

Reply:
Also checked: Cascade OnUpdate is default. So all we realistically have to do is set the id to the GUID function in the migration